src/Extensions/TopPageElementExtension.php

```diff
@@ -55,7 +55,7 @@ class TopPageElementExtension extends Extension
     private $fixedTopPageID = 0;
 
     /**
-     * Extension point in @see DataObject::onAfterWrite()
+     * Extension point in {@see DataObject::onAfterWrite()}
```

Comment:
This is the correct syntax - my IDE just treated it as raw text and not an inline annotation without the curlies.